pkgs/development/python-modules/fenics/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, lib
+, fetchurl
+, buildPythonPackage
+, isPy27
+, boost
+, cmake
+, mpi4py
+, pybind11
+, dijitso
+, numpy
+, sympy
+, ufl
+, pkgconfig
+, ffc
+, dolfin
+}:
+
+buildPythonPackage rec {
+  pname = "fenics";
+  inherit (dolfin) version;
+  disabled = isPy27;
```

Comment:
```suggestion
```

python 2 packages got split off.

Reply:
Meaning it will no longer be supported and is considered an edge case?